torch/nested/_internal/nested_tensor.py

```diff
@@ -11,11 +11,13 @@
 _tensor_symint_registry = WeakTensorKeyDictionary()
 
 
-def get_tensor_symint(tensor, *, coeff=1):
+def get_tensor_symint(tensor, *, coeff=1, sum_offsets=None):
     global _tensor_id_counter
     tensor_symint = _tensor_symint_registry.get(tensor)
     if tensor_symint is None:
-        tensor_symint = torch._C._get_singleton_int(_tensor_id_counter, coeff)
+        tensor_symint = torch._C._get_singleton_int(
+            _tensor_id_counter, coeff, tensor, get_nt_dummy(), sum_offsets
```

Comment:
any way we can avoid storing the dummy on the singleton since we have a single global dummy per class?